contrib/verify-commits/README.md

```diff
@@ -3,7 +3,7 @@ Tooling for verification of PGP signed commits
 
 This is an incomplete work in progress, but currently includes a pre-push hook
 script (`pre-push-hook.sh`) for maintainers to ensure that their own commits
-are PGP signed (nearly always merge commits), as well as a script to verify
+are PGP signed (nearly always merge commits), as well as a python 3 script to verify
```

Comment:
Nit: upper case Python. I'm a bit surprised `#!/usr/bin/env python3` at the top of that script wouldn't complain when it's run with Python 2.

Reply:
I updated the capitalization. Played around with the python version support but found out that it's crashing immediately upon interpreting the script, so it's not even getting to attempting execution. Thus I don't think there's any way to improve the current state of the script without trying to rewrite all of the logging calls that cause the interpreter to blow up.